scripts/jenkins/qa_devstack.sh

```diff
@@ -103,6 +103,9 @@ function h_setup_devstack {
     if ! getent group nobody >/dev/null; then
         $zypper in 'group(nogroup)'
     fi
+    # pycparser got pulled in via recommends, but devstack later needs to be
+    # able to update it via pip.
+    $zypper -n remove python3-pycparser
```

Comment:
can't we do `--no-recommends` instead?

Reply:
We could, but explicitly ensuring pycparser is not installed matches the intent more closely. Whether it got installed as a recommend or some other means is not that important.

Reply:
ok

Reply:
NIT: "-n" is already part of $zypper